scripts/gh-predeploy.js

```diff
@@ -303,11 +306,32 @@ function prepareToDeployPlugins(done) {
   });
   return gulp.parallel(folders.map(function (folder) {
     return function preDeployPlugin() {
-      return preparePlugin(folder);
+      return preparePlugin(folder, false);
     };
   }))(done);
 }
 
+/**
+ * Prepare plugins for deployment to gh-pages.
+ *
+ * For each plugin, copy relevant files to the gh-pages directory.
+ * @param {Function} done Completed callback.
+ * @return {Function} Gulp task.
+ */
+function prepareLocalPlugins(done) {
```

Comment:
This looks like it duplicates a lot of code from `prepareToDeplyPlugins`. Could these be unified using a subprocedure or another level of closure?

Same for examples.

Reply:
Beka and I took at look at this but couldn't find a good way to unify the functions that was actually more readable.